src/utils/helpers.js

```diff
@@ -72,11 +72,17 @@ export const getFilenameFromPath = (path) => {
 /**
  * Returns extension from the given path or filename
  * @param {String} path or filename
- * @return {String} extension
+ * @return {String} extension or nil
  */
 export const getExtensionFromPath = (path) => {
   if (!path) return '';
-  return path.substring(path.lastIndexOf('.') + 1);
+  const index = path.lastIndexOf('.');
+
+  if (index >= 0) {
```

Comment:
Yeah but since we are checking the last `.` here, dotFiles with no extension will get in the if statement. For example, `.babelrc`; It has no extension but `index` variable will be 0.